movement_primitives/dmp/_dmp.py

```diff
@@ -380,6 +380,12 @@ class DMP(WeightParametersMixin, DMPBase):
         is changed and the trajectory is scaled by interpolating between
         the old and new scaling of the trajectory.
 
+    alpha_y : float, list with length n_dims, or array with shape (n_dims,), optional (default: 25.0)
```

Comment:
I would just write `array-like, shape (n_dims,)`

Reply:
Addressed in [7e5d15e](https://github.com/dfki-ric/movement_primitives/pull/59/commits/7e5d15ea3cd1587d8f2bbca949f1a8385325a8f8)